website/docs/reference/dbt-jinja-functions/model.md

```diff
@@ -15,9 +15,52 @@ For example:
 {% endif %}
 ```
 
-To view the contents of `model` for a given model, use [log()](/reference/dbt-jinja-functions/log)
-to print the full contents:
+To view the contents of `model` for a given model:
+
+<Tabs>
+
+<TabItem value="cli" label="CLI">
+
+If you're using the CLI, use [log()](/reference/dbt-jinja-functions/log) to print the full contents:
 
 ```jinja
 {{ log(model, info=True) }}
 ```
+  
+ </TabItem>
+ 
+ <TabItem value="ide" label="dbt Cloud IDE">
+   
+ If you're using the dbt Cloud IDE, compile the following to print the full contents: <br /><br />
+
+ ```jinja
+{{ model | tojson(indent = 4) }}
+```
+   
+</TabItem>
+
+</Tabs>
+
+## Model structure and JSON schema
+
+To view the structure of `models` and their definitions:
+- Refer to [dbt JSON Schema](https://schemas.getdbt.com/) for describing and consuming dbt generated artifacts
+- Select the corresponding manifest version under **Manifest**. For example if you're on dbt v1.3, then you would select Manifest v7
+  * The `manifest.json` version number isn't directly related to your dbt version, so you_must_ use the correct `manifest.json` version for your dbt version. To find the correct `manifest.json` version, refer to [Manifest](/reference/artifacts/manifest-json) and select the dbt version on the top navigation (such as `v1.5`). This will help you find out which tags are associated with your model. 
```

Comment:
```suggestion
  * The `manifest.json` version number is related to (but not _equal_ to) your dbt version, so you _must_ use the correct `manifest.json` version for your dbt version. To find the correct `manifest.json` version, refer to [Manifest](/reference/artifacts/manifest-json) and select the dbt version on the top navigation (such as `v1.5`). This will help you find out which tags are associated with your model.
```